builtin/logical/mongodb/path_config_lease.go

```diff
@@ -0,0 +1,107 @@
+package mongodb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"ttl": &framework.FieldSchema{
+				Type:        framework.TypeString,
```

Comment:
You should use `framework.TypeDurationString` here. This will automatically handle values both sent as integers (e.g. `300`) and with a suffix (`300s` or `72h`) and doesn't require you to call `time.ParseDuration`. It will *always* return an int when you call `d.Get` so you can safely just read that integer value.